ui/src/app/views/research-purpose.tsx

```diff
@@ -0,0 +1,153 @@
+      }
+    }
+
+    async publishUnpublishWorkspace(publish: boolean) {
+      this.setState({publishing: true});
+      try {
+        if (publish) {
+          await workspacesApi()
+            .publishWorkspace(this.props.workspace.namespace, this.props.workspace.id);
+        } else {
+          await workspacesApi()
+            .unpublishWorkspace(this.props.workspace.namespace, this.props.workspace.id);
+        }
+      } catch (error) {
+        console.error(error);
+      } finally {
+        this.setState({publishing: false});
+      }
+    }
+
+    render() {
+      const {workspace, profileState: {profile}} = this.props;
+      const {publishing, workspacePermissions} = this.state;
+      const selectedResearchPurposeItems = this.getSelectedResearchPurposeItems();
+      const rpItemsHalfLen = sliceByHalfLength(selectedResearchPurposeItems);
+      return <FadeBox style={{margin: '1rem', width: '98%'}}>
+        <div style={styles.mainHeader}>Research Purpose
+          <Clickable disabled={!workspacePermissions.canWrite}
+                     data-test-id='edit-workspace'
+                     onClick={() => navigate(
+                       ['workspaces',  workspace.namespace, workspace.id, 'edit'])}>
+            <EditComponentReact enableHoverEffect={true}
+                                disabled={!workspacePermissions.canWrite}
+                                style={{marginTop: '0.1rem'}}/>
+          </Clickable>
+        </div>
+        <div style={styles.sectionText}>{ResearchPurposeDescription}</div>
+        <div style={styles.sectionHeader}>Primary purpose of the project</div>
+        <TwoColPaddedTable header={false} style={{marginTop: '0.3rem'}}
+                           contentLeft={selectedResearchPurposeItems.slice(0, rpItemsHalfLen)}
+                           contentRight={selectedResearchPurposeItems.slice(rpItemsHalfLen)}/>
+        <div style={styles.sectionHeader}>
+          Reason for choosing <i>All of Us</i> data for your investigation</div>
+        <div style={styles.sectionText}>{workspace.researchPurpose.reasonForAllOfUs}</div>
+        <div style={styles.sectionHeader}>Area of intended study</div>
+        <div style={styles.sectionText}>{workspace.researchPurpose.intendedStudy}</div>
+        <div style={styles.sectionHeader}>Anticipated findings from this study</div>
+        <div style={styles.sectionText}>{workspace.researchPurpose.anticipatedFindings}</div>
+        {workspace.researchPurpose.population && <div style={{marginBottom: '1rem'}}>
+          <div style={styles.sectionHeader}>Population area(s) of focus</div>
+          <TwoColPaddedTable header={false} style={{marginTop: '0.3rem'}}
+                             contentLeft={this.getSelectedPopulationsSlice(true)}
+                             contentRight={this.getSelectedPopulationsSlice(false)}/>
+        </div>
+        }
+        {profile.authorities.includes(Authority.FEATUREDWORKSPACEADMIN) &&
```

Comment:
I think this should be in the workspace about page, and isn't necessarily tied to research purpose?